src/-lib/html-normalizer.js

```diff
@@ -80,27 +80,33 @@ function isAbsoluteURL(url) {
 function reconcilePaths(startPath, updatePath) {
   if (/^\//.test(updatePath)) {
     return updatePath;
-  } else {
-    const pathParts = [...pathDir(startPath).split('/'), ...updatePath.split('/')];
-
-    // Ensure there aren't any "double slashes" in the middle of the path.
-    return pathParts
-      .filter((part, index, arr) => part !== '' || index === 0 || index == arr.length - 1)
-      .join('/');
   }
+
+  const pathParts = [
+    ...pathDir(startPath).split('/'),
+    ...updatePath.split('/')
+  ];
+
+  // Ensure there aren't any "double slashes" in the middle of the path.
+  return pathParts
+    .filter(
+      (part, index, arr) =>
+        part !== '' || index === 0 || index === arr.length - 1
+    )
+    .join('/');
 }
 
 function pathDir(path) {
-  return path.replace(/[^\/]+$/, '');
+  return path.replace(/[^/]+$/, '');
```

Comment:
Just checking that this regex change is intentional...

Reply:
I don't there's any actual change here, prettier just removes escapes from characters that don't need them

Reply:
Confirm!